docs/buildspecs/e4s_testsuite.rst

```diff
@@ -0,0 +1,43 @@
+.. Note:: Please see :ref:`tutorial_setup` before you proceed with this section
+
+.. _buildtest_e4s_testsuite:
+
+Buildtest E4S Testsuite Integration
+===================================
+
+The `E4S Testsuite <https://github.com/E4S-Project/testsuite/>`_ is a collection of lightweight tests assembled for over 100 
+software products provided by Extreme Scale Scientific Software Stack `(E4S) <https://e4s.io>`_ deployments. 
+E4S is a spack based software stack but because not all spack packages provide internal testing functionality it 
+is useful to have an alternative framework for testing deployed software.
```

Comment:
okay in that case can you rewrite this sentence right now i see two distinct sentences 
- `E4S is a spack based software stack.` 
- `but because not all spack packages ...`

I think you should separate the sentence and point out the main purpose of E4S Testsuite, what its purpose and why it exists.

I can think of a few reasons why it exists

1. Not all spack packages have tests
2. `spack test` was introduced in v0.16 so no way of testing older stacks
3. E4S Testsuite is more flexible with updating tests, while spack tests are static within a spack version.